spec/controller_extension_spec.rb

```diff
@@ -0,0 +1,79 @@
+# spec/readyset/controller_extension_spec.rb
+
+require 'rails_helper'
+require 'ready_set/controller_extension'
+
+RSpec.describe ReadySet::ControllerExtension, type: :controller do
+  # Global Set-up
+  controller(ActionController::Base) do
+    # NOTE: Not required as it's setup by the ReadySet Railtie
+    include ReadySet::ControllerExtension
```

Comment:
Can we remove this line? It looks like the specs still pass without it, I think because combustion properly invokes our action_controller initializer